test/e2e/index.spec.js

```diff
@@ -0,0 +1,29 @@
+import { resolve } from 'path'
```

Comment:
I think this may be extraneous for now. It may be a useful blueprint for e2e testing, but I think it can be deleted (or safely stashed away with git stash)